File: week01/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);

    int x = 1;

    for (int i = 0; i < n; i++)
    {
        for (int y = 0; y < n - x; y++)
        {
            printf(" ");
        }
        for (int j = 0; j < x; j++)
        {
            printf("#");
        }
        x = x + 1;
        printf("\n");
    }
}

File: week06/sentimental-mario-more/mario.py
from cs50 import get_int

# Prompt the user for the half-pyramid's height
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Generate the desired half-pyramids
counter = 1

for i in range(height):
    for x in range(height - counter):
        print(" ", end="")
    for y in range(counter):
        print("#", end="")
    print("  ", end="")
    for m in range(counter):
        print("#", end="")
    # for n in range(height - counter):
        # print(" ", end="")
    print("")
    counter += 1